pom.xml

```
@@ -55,6 +55,13 @@
             <version>${bootstrap.version}</version>
         </dependency>
 
+        <dependency> 
+            <groupId>com.googlecode.mp4parser</groupId> 
+            <artifactId>isoparser</artifactId> 
+            <version>1.0.1</version>
```

Actually, this should be 1.0.2

```suggestion
            <version>1.0.2</version>
```